src/main/java/org/maxgamer/quickshop/QuickShop.java

```diff
@@ -808,7 +808,7 @@ public void onEnable() {
         Bukkit.getPluginManager().registerEvents(internalListener, this);
         if (isDisplay() && DisplayItem.getNowUsing() != DisplayType.VIRTUALITEM) {
             displayWatcher = new DisplayWatcher(this);
-            new DisplayBugFixListener(this).register();
+//            new DisplayBugFixListener(this).register();
```

Comment:
Why disable this?

Reply:
It designed for ArmorStandDisplay and it now deleted

Reply:
It designed for ArmorStandDisplay and it now deleted